ci/run.sh

```diff
@@ -36,3 +36,12 @@ cargo_test() {
 
 cargo_test
 cargo_test "--release"
+
+case ${TARGET} in
+    x86*)
+        RUSTFLAGS="${RUSTFLAGS} -C target-feature=+avx"
+        cargo_test "--release"
```

Comment:
This'll somehow need to disable all the `assert_instr` tests as well as anything beneath avx is likely to generate different instructions with avx enabled